### PR TITLE
Add table management section

### PR DESCRIPTION
**Pull request by erjohnson**
* Create table management top level section
* Move creating and activating table page to new section
* Move global expiry page to new section
* Add per table expiry page to new section
* Update version history frontmatter across versions to reflect changes in location
* Update links pointing to moved pages

### CODE REVIEW

File: content/riak/ts/1.6.0/table-management.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+[activating]: /riak/ts/1.6.0/table-management/creating-activating/
+[global expiry]: /riak/ts/1.6.0/table-management/global-object-expiration/
+[table expiry]: /riak/ts/1.6.0/table-management/per-table-object-expiration/
+[mdc]: /riak/ts/1.6.0/configuring/mdc/
+[writing]: /riak/ts/1.6.0/using/writingdata/
+
+After [planning][plan] your Riak TS table:
+
+1. [Create and activate][activating] your Riak TS table. (You'll need `sudo` and `su` access for this step.)
+2. [Enable global object object expiration][global expiry] if you plan on setting up an object retention policy. If you are an Enterprise user, see the [per table object expiration][table expiry] instructions to enable object retention on per table basis.
```
Comment (laurenrother) [inline content/riak/ts/1.6.0/table-management.md]: "on a per table basis".

File: content/riak/ts/1.6.0/table-management/per-table-object-expiration.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+[create table]: /riak/ts/1.6.0/table-management/creating-activating/
+[create table with]: /riak/ts/1.6.0/table-management/creating-activating/#using-with
+
+By default, LevelDB keeps all of your data. But Riak TS allows you to configure object expiration (`expiry`) or [time to live (TTL)][ttl] for your data on a [global][global expiry] or per table basis.
+
+Expiration is disabled by default, but enabling it lets you expire older objects to reclaim the space used or purge data with a limited time value.
+
+## Enabling Expiry
+
+To enable object expiry for global or per table use, add the `leveldb.expiration` setting to your riak.conf file:
```
Comment (laurenrother) [inline content/riak/ts/1.6.0/table-management/per-table-object-expiration.md]: So are these instructions also in the global expiration page?

File: content/riak/ts/1.6.0/table-management/per-table-object-expiration.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+## Expiry Table Properties
+
+|Property Name|Values|
+|---|---|
+|expiration|`enabled` / `disabled`|
+|defaut_time_to_live|`unlimited` or a duration string|
+|expiration_mode|`use_global_config` / `per_item` / `whole_file`|
+
+Each table can have one or more of the properties listed above. If any properties are omitted on the table, the property values in riak.conf will be used. If the properties are not set within riak.conf, the default values will be used.
```
Comment (laurenrother) [inline content/riak/ts/1.6.0/table-management/per-table-object-expiration.md]: Where do I find what the default values are?

File: content/riak/ts/1.6.0/table-management/per-table-object-expiration.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+By default, LevelDB keeps all of your data. But Riak TS allows you to configure object expiration (`expiry`) or [time to live (TTL)][ttl] for your data on a [global][global expiry] or per table basis.
+
+Expiration is disabled by default, but enabling it lets you expire older objects to reclaim the space used or purge data with a limited time value.
+
+## Enabling Expiry
+
+To enable object expiry for global or per table use, add the `leveldb.expiration` setting to your riak.conf file:
+
+```riak.conf
+leveldb.expiration = on
```
Comment (laurenrother) [inline content/riak/ts/1.6.0/table-management/per-table-object-expiration.md]: So, and this is just me being confused, this turns expiration on for all tables? Is there a way to enable expiration for a specific table after it has been created?

File: content/riak/ts/1.6.0/table-management/per-table-object-expiration.md (hunk 2 above)
Comment (erjohnson) [inline content/riak/ts/1.6.0/table-management/per-table-object-expiration.md]: They are, expiration still has to be enabled afaik in the conf file

**Comment by laurenrother** [comment]
Looks awesome @erjohnson ! 🎉 👍

**Pull request merged**